scripts/themer

```diff
@@ -84,7 +84,7 @@ def get_filters():
         return get_component(color, 2)
 
     def to_float(num):
-        return num / 255
+        return num / 255.
```

Comment:
Ok I'll do that instead. On an unrelated note, is `themer` deterministic? I've tried creating more colorschemes (with the same wallpaper) for testing but they come out with different colors.